src/loaders/lottie/tvgLottieModel.h

```diff
@@ -637,8 +637,15 @@ struct LottieTransform : LottieObject
     {
         LottieFloat x = 0.0f;
         LottieFloat y = 0.0f;
+        bool apply = true;
```

Comment:
Please consider to remove this additional property. I don't see its really necessary.